packages/happy-dom/src/nodes/html-option-element/HTMLOptionsCollection.ts

```diff
@@ -34,14 +34,7 @@ export default class HTMLOptionsCollection
 	 * @returns SelectedIndex.
 	 */
 	public get selectedIndex(): number {
-		for (let i = 0; i < this.length; i++) {
-			const item = this[i];
-			if (item instanceof HTMLOptionElement && item.selected) {
```

Comment:
I'm a bit concerned that by removing this behavior you're going to break some things. If the HTML was the following:

```html
<select>
	<option>1</option>
	<option selected>2</option>
</select>
```
The browser says selectedIndex is 1, but with this change it'll wrongly be 0, won't it?

Reply:
@IGx89 Yes you are right. I will fix it now directly. I wanted to correct the `selectedIndex` behavior as the current behavior does not follow the spec or the browser behavior, but I missed the "selected" attribute.